<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(255, 76, 76);
}
QGroupBox{
	background-color: rgb(99, 98, 174);
}
QPushButton#btn1{
	color: rgb(32, 74, 135);
	background-color: rgb(238, 238, 236);
	border-radius:8px;
	font: 75 italic 15pt &quot;Z003&quot;;
}
QPushButton#btn2{
	color: rgb(32, 74, 135);
	background-color: rgb(238, 238, 236);
	border-radius:8px;
	font: 75 italic 15pt &quot;Z003&quot;;
}
QPushButton#btn3{
	color: rgb(32, 74, 135);
	background-color: rgb(238, 238, 236);
	border-radius:8px;
	font: 75 italic 15pt &quot;Z003&quot;;
}
QPushButton#btn4{
color: rgb(32, 74, 135);
	background-color: rgb(238, 238, 236);
	border-radius:8px;
	font: 75 italic 15pt &quot;Z003&quot;;
}
QPushButton#btn5{
color: rgb(32, 74, 135);
	background-color: rgb(238, 238, 236);
	border-radius:8px;
	font: 75 italic 15pt &quot;Z003&quot;;
}
QPushButton#btn6{
	color: rgb(32, 74, 135);
	background-color: rgb(238, 238, 236);
	border-radius:8px;
	font: 75 italic 15pt &quot;Z003&quot;;
}
QPushButton#btn7{
color: rgb(32, 74, 135);
	background-color: rgb(238, 238, 236);
	border-radius:8px;
	font: 75 italic 15pt &quot;Z003&quot;;
}
QPushButton#btn8{
	color: rgb(32, 74, 135);
	background-color: rgb(238, 238, 236);
	border-radius:8px;
	font: 75 italic 15pt &quot;Z003&quot;;
}
QPushButton#btn9{
color: rgb(32, 74, 135);
	background-color: rgb(238, 238, 236);
	border-radius:8px;
	font: 75 italic 15pt &quot;Z003&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>1</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item row="1" column="2">
        <widget class="QPushButton" name="btn6">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn2">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn3">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn8">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn4">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn1">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn7">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn5">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btn9">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="placeholderText">
            <string>Nama pemain</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnadd">
           <property name="styleSheet">
            <string notr="true">border-radius:4px;
font: 75 10pt &quot;Kalimati&quot;;
color: rgb(99, 209, 70);
background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Tambah Pemain</string>
           </property>
           <property name="icon">
            <iconset resource="source.qrc">
             <normaloff>:/Icon/1.png</normaloff>:/Icon/1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnReset">
           <property name="styleSheet">
            <string notr="true">border-radius:4px;
font: 75 10pt &quot;Kalimati&quot;;
color: rgb(99, 209, 70);
background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Reset Game</string>
           </property>
           <property name="icon">
            <iconset resource="source.qrc">
             <normaloff>:/Icon/2.png</normaloff>:/Icon/2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnpengaturan">
           <property name="styleSheet">
            <string notr="true">border-radius:4px;
font: 75 10pt &quot;Kalimati&quot;;
color: rgb(99, 209, 70);
background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Pengaturan</string>
           </property>
           <property name="icon">
            <iconset resource="source.qrc">
             <normaloff>:/Icon/3.png</normaloff>:/Icon/3.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="player1">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Yrsa</family>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="p1">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="player2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Yrsa</family>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="p2">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lblVolume">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;Noto Sans CJK SC&quot;;</string>
           </property>
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="volume">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
